### DISCUSSION TITLE
Release 5.5

### DISCUSSION THREAD

**Discussion by wilbit** [nhibernate/nhibernate-core#3436] [General] [answered]
When will it be released officially? Any dates?

**Comment by gudmundurm** [comment]
@hazzik & @fredericDelaporte any idea when?

**Comment by fredericDelaporte** [comment, marked as answer]
We do not have roadmaps. But since there is nothing more to be finished in the milestone currently, why not releasing it "soon".

@hazzik, @bahusoid, any thought?

**Reply by fredericDelaporte** [reply]
See #3446.
